pages/coq-team.html

```diff
@@ -0,0 +1,486 @@
+    <div class="position">Inria Researcher in the Stamp Team, Sophia-Antipolis, France</div>
+    <div class="role">RM: 8.13</div>
+    <div class="components">General Maintenance, STM, ssreflect, VSCoq, Mathematical Components, Coq-Elpi, Hierarchy Builder</div>
+  </div>
+  <div class="person">
+    <div class="picture"><img src="/files/team/Zimmi48.jpg" alt="Théo Zimmermann" width="200px"/></div>
+    <div class="name"><a href="https://www.theozimmermann.net/en/">Théo Zimmermann</a></div><div class="nickname">Zimmi48</div>
+    <div class="position">Post-Doctoral Student in the pi.r2 Team, Paris, France</div>
+    <div class="role">RM: 8.7, 8.8, 8.12, coq-community manager</div>
+    <div class="components">Documentation, Community Management, devtools, <verb>coqbot</verb></div>
+  </div>
+</div>
+
+
+<div class="team">
+  <div class="team-name"><h1>Maintainers</h1></div>
+  <div class="person">
+    <div class="picture"><img src="/files/team/fbesson.jpg" alt="Fréderic Besson" width="200px"/><br/></div>
+    <div class="name"><a href="http://people.rennes.inria.fr/Frederic.Besson/">Frédéric Besson</a></div>
+    <div class="nickname">fbesson</div>
+    <div class="position">Inria Researcher in the Celtique Team, Rennes, France</div>
+    <div class="components">Decision Procedures</div>
+  </div>
+  <div class="person">
+    <div class="picture">
+      <img src="/files/team/cyril-cohen.png" alt="Cyril Cohen" width="200px"/></div>
+    <div class="name"><a href="https://perso.crans.org/cohen/">Cyril Cohen</a></div><div class="nickname">CyrilCohen</div>
+    <div class="position">Inria Researcher in the Stamp Team, Sophia-Antipolis, France</div>
+    <div class="components">Opam, Ssreflect, Nix, Mathematical Components, Hierarchy Builder</div>
+  </div>
+
+  <div class="person">
+    <div class="picture"><img src="/files/team/PierreCorbineau.jpg" alt="Pierre Corbineau" width="170px"/></div>
+    <div class="name"><a href="http://cedric.cnam.fr/~courtiep/">
+      Pierre Corbineau</a></div><div class="nickname">PierreCorbineau</div>
+    <div class="position">Assistant Professor at VERIMAG, Grenoble, France</div>
+    <div class="components">Decision Procedures (firstorder, cc, rtauto)</div>
+  </div>
+  
+  <div class="person">
+    <div class="picture"><img src="/files/team/matafou.png" alt="Pierre Courtieu" width="200px"/></div>
+    <div class="name"><a href="http://cedric.cnam.fr/~courtiep/">Pierre Courtieu</a></div><div class="nickname">Matafou</div>
+    <div class="position">Assistant Professor at CNAM, Paris, France</div>
+    <div class="components">Function, Proof General</div>
+  </div>
+  <div class="person">
+    <div class="picture"><img src="/files/team/nopic.png" alt="Jim Fehrle" width="200px"/></div>
+    <div class="name">Jim Fehrle</div><div class="nickname">jfehrle</div>
+    <div class="position">Software Engineer in Menlo Park, CA, USA</div>
+    <div class="components">Documentation</div> <!-- LTac debugger -->
```

Comment:
```suggestion
    <div class="components">Documentation, Proof diffs</div> <!-- LTac debugger -->
```

Reply:
I added Ltac debugger too, as it's in 8.14 now.